developer-docs/dev_setup.md

```diff
@@ -1,215 +1,145 @@
 
-Following this you should have some information printed to the console that follows this example:
+You can follow the instructions to install `kubectl` from the [Kubernetes website]. You can verify the installation is successful with:
 
 ```bash
-You need a passphrase to unlock the secret key for
-user: "John Smith <john.smith@xxxxxxx.xxxxxxxxxx.xxx.xx>"
-4096-bit RSA key, ID AAAAAAAA, created 2000-01-01 (main key ID BBBBBBBB)
-
-gpg: encrypted with 4096-bit RSA key, ID BBBBBBBB, created 2000-01-01
-      "John Smith <john.smith@xxxxxxx.xxxxxxxxxx.xxx.xx>"
-XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX
+$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.3", GitCommit:"2c2fe6e8278a5db2d15a013987b53968c743f2a1", GitTreeState:"clean", BuildDate:"2017-08-03T07:00:21Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
 ```
+#### Drone
 
-The last printed line in the console, below the 2nd iteration of your name and 365 email address,
-is the token you need to include in your kubectl config file. Once you have included it in the relevant place in the config file, you should be all setup! ^_^
+Drone CLI can be found from the [Drone CI Docs]. Instructions installing on multiple operating systems are stated on the webpage.
 
-### Use kubectl
-
-You can verify the installation is successful with:
+Verify that installation the installation is successful:
 
 ```bash
-$ kubectl get pods
+$ drone --version
+drone version 0.8.0
 ```
 
-> **Please note** that you need to have your VPN running to talk to the kubernetes cluster.
+#### KD
 
-You should get an empty reply with just some column headers. The config file by default looks only at the *dev-induction* namespace.
-To look at other namespace you can, for example do:
+`kd` is minimalistic kubernetes resource deployment tool that we use. You can get the latest version of kd on its [releases page].
 
+Verify that installation is successful:
 ```bash
-$ kubectl --namespace=my-namespace get pods
+$ kd -v
+kd version v0.7.0
 ```
-You can also edit your kubernetes config, either by editing the file directly or by running:
-
-```bash
-$ kubectl config
-```
-
-## Install minikube
 
-You can follow the instruction to install `minikube` from the [Minikube website](https://github.com/kubernetes/minikube/releases).
+## Connecting to ACP VPN
 
-> **Please note** that you have to have kubectl installed before you can run minikube.
+Once you've got an Office 365 Account you can now navigate to [Access ACP] and login with your Office 365 account by clicking on the 365 link on the right.
 
-If the installation is successful, you can run minikube locally with:
-
-```bash
-$ minikube start
-Starting local Kubernetes cluster...
-Kubectl is now configured to use the cluster.
-```
-
-### Setting up minikube ingress
-
-```bash
-$ minikube addons enable ingress
-```
+Once you're logged in download the VPN profile called **"ACP Platform (Ops, Dev, Ci, Test)"** from under VPN Profiles.
```

Comment:
Please put this after openvpn section above and ask them to make sure they can resolve the Hub.